.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file should not be incuded.

Reply:
it is generated by vscode, just remove it